src/commands/beer_style.rs

```diff
@@ -48,39 +48,25 @@ pub fn add_subcommand<'a, 'b>() -> App<'a, 'b> {
         )
 }
 
+fn matches_to_criteria<'a>(matches: &ArgMatches<'a>) -> Criteria {
+    Criteria {
+        og: matches.value_of("og").map(|value| value.parse().unwrap()),
```

Comment:
Ain't these exact matches? Idea was to find beer styles where input of for example OG is between the beer styles min and max OG.

Reply:
This function is just responsible for parsing the command line options into numbers and returning a `Criteria` struct - the actual range matching logic still exists in the `utils/beer_styles.rs` file, and behaves the same as before with ranges:

https://github.com/drodil/rustybeer/blob/d336213ad27e52162b936de4a872dcf1f5a8fc2d/src/utils/beer_styles.rs#L42